### PR TITLE
chore: update module from template

### PR DESCRIPTION
**Pull request by zied-elouaer**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -20,18 +17,126 @@ jobs:
...
+        # actions/setup-go@v3.2.0 -> b22fbbc2921299758641fab08929b4ac52b32923
+        uses: actions/setup-go@b22fbbc2921299758641fab08929b4ac52b32923
+        with:
+          go-version: 1.17.6
+
+      - name: Set up Terramate
+        uses: giantswarm/install-binary-action@e97402b0aa95939ddba5b36aa77f8abc58a37fd5
+        with:
+          binary: terramate
+          version: 0.1.35
```
Comment (waxb) [inline .github/workflows/tests.yml:46]:
```suggestion
          version: 0.1.41
```

File: README.tfdoc.hcl [hunk 2]
```diff
@@ -239,8 +239,8 @@ section {
         END
         readme_example = <<-END
           iam = [{
-            role    = "roles/iam.serviceAccountUser"
-            members = ["user:member@example.com"]
+            role               = "roles/iam.serviceAccountUser"
```
Comment (waxb) [inline README.tfdoc.hcl:242]: did `fmt` do this?

**Review by waxb — CHANGES_REQUESTED** [review]
changelog should reflect that the module drops support for `google` provider `3.x`

**Comment by waxb** [comment]
please update the pr title

**Comment by waxb** [comment]
please refresh `.gitignore` also

**Review by waxb — APPROVED** [review]
lgtm

**Review by soerenmartius — APPROVED** [review]
lgtm

**Review by mariux — APPROVED** [review]
lgtm

**Pull request merged**